07/hangman/hangman_game.py

```diff
@@ -0,0 +1,33 @@
+from hangman_settings import vytvor_retezec
+from hangman_pismeno import pismeno
+from hangman_score import score
+from hangman_pismeno_ve_slove import dosazeni_pismene
+from random import choice
+
+
+def hangman_game(seznam):
+    pokus = 0
+    print()
+    print('Hra: OBĚŠENEC')
+    print('Uhádni písmeno ve slově a poskládej tak slovo')
+    slovo = choice(seznam)
+    retezec = vytvor_retezec(slovo)
+    print(retezec)
+    print(slovo)
+
+    while True:
+        pismenko_k_dosazeni = pismeno()
```

Comment:
Funkcia `pismeno()` je za mňa asi zbytočná komplikácia. Namiesto tohoto riadku by ti stačilo toto  `pismeno = input("Hádej písmeno ")` a tá funkcia sú za mňa zbytočné 3 riadky. Akurát to zhoršuje čitateľnosť, pretože sa musíš dívať inam, do iného súboru čo tá funkcia robí.

Reply:
ok :)